pkg/registry/core/rest/storage_core.go

```diff
@@ -451,7 +512,50 @@ func (c LegacyRESTStorageProvider) newServiceIPAllocators() (registries serviceI
 	return
 }
 
-func (p LegacyRESTStorageProvider) GroupName() string {
+var _ genericapiserver.PostStartHookProvider = &legacyProvider{}
+
+func (p *legacyProvider) PostStartHook() (string, genericapiserver.PostStartHookFunc, error) {
```

Comment:
does this error crash the apiserver?
it should to maintain the behavior

Reply:
it does

Reply:
It does:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/hooks.go#L201-L204